src/lib/validator-dob.ts

```diff
@@ -30,3 +30,31 @@ const parse = (input: string) => {
   const strippedInput = input.replace(/ /ig, '');
   return DateTime.fromISO(strippedInput);
 };
+
+export class ValidatorAge implements IValidator {
+  private dobValidator = new ValidatorDateOfBirth();
+
+  private trimSpace(input: string): string {
+    return input.replace(/\s/g, '');
+  }
+
+  isValid(input: string): boolean {
+    const age = Number(this.trimSpace(input));
+    return (!isNaN(age) && age > 18 && age < 200) || (this.dobValidator.isValid(input));
```

Comment:
can you make a 10y/o user with a date input but not an int? at minimum we should inforce this for all inputs.

why `18`? dob has 0 hours.... where is this requirement from?  i think doubt cht-user-management should be the thing that stops an 18yo CHW from entering the eCHIS system....  should it be 10 or something which validates sanity but isn't too controversial?

Reply:
I've enforced that dates should be less than the current now. I used 18 on the assumption that anyone at or below is a minor. Until there's an explicit request from a partner for the minimum then i think this should be enough